docker/production/Dockerfile.website

```diff
@@ -47,13 +41,25 @@ RUN dnf -y install python3-fastapi \
                    python3-koji \
                    python3-copr \
                    koji \
+                   htop \
+                   btop \
+                   make \
+                   wget \
+                   net-tools \
+                   iputils \
+                   vim \
+                   mlocate \
+                   jq \
+                   fpaste \
+                   git \
+                   python3-ipdb \
+                   findutils \
     && dnf clean all \
     && mkdir -p /src/{frontend,backend}
+
 COPY --from=0 /src/frontend/public /src/frontend/public
 COPY --from=0 /src/backend /src/backend
 
-# According to the documentation, gunicorn is a valid production server
-# https://www.uvicorn.org/deployment/
```

Comment:
Why remove comments like these? I think it was helpful.

Reply:
some leftover change, thanks for noticing